packages/core/core-flows/src/payment-collection/steps/delete-payment-sessions.ts

```diff
@@ -0,0 +1,101 @@
+import { ModuleRegistrationName } from "@medusajs/modules-sdk"
+import {
+  IPaymentModuleService,
+  Logger,
+  PaymentSessionDTO,
+} from "@medusajs/types"
+import { ContainerRegistrationKeys } from "@medusajs/utils"
+import { StepResponse, createStep } from "@medusajs/workflows-sdk"
+
+interface StepInput {
+  ids: string[]
+}
+
+// Note: This workflow should not be used alone as it doesn't consider a revert
```

Comment:
```suggestion
// Note: This step should not be used alone as it doesn't consider a revert
```

Reply:
Is this comment relevant?

Reply:
yes, because the next step in the workflow will revert this if something goes wrong.